aws/components/lb/setup.ftl

```diff
@@ -297,7 +297,9 @@
 
                 [/#list]
 
-                [#local listenerRuleConditions += getListenerRuleHostCondition(fqdn) ]
+                [#if fqdn?has_content]
+                    [#local listenerRuleConditions += getListenerRuleHostCondition(fqdn) ]
+                [/#if]
```

Comment:
I think this need to be an exception rather than a bypass. The host filter option is more a security service so needs to be explicit rather still allow it to happen.

Reply:
So a warning message (but still allow it through), or a fatal if no domainRedirectRules were found, or something else?

Reply:
If hostFilter is enabled and there is no fqdn then it should be a fatal. We can't do what the user has requested

Reply:
Cool, done